include/deal.II/numerics/data_postprocessor.h

```diff
@@ -528,7 +600,8 @@ class DataPostprocessor : public Subscriptor
  *
  * All derived classes have to do is implement a constructor and overload
  * either DataPostprocessor::evaluate_scalar_field() or
- * DataPostprocessor::evaluate_vector_field().
+ * DataPostprocessor::evaluate_vector_field() as discussed in the
+ * DataPostprocessor class's document.
```

Comment:
Don't you spell this like this:
```suggestion
 * DataPostprocessor class' document.
```

Reply:
No, in the English language the 's' after the apostrophe is only dropped if you the 's' before the apostrophe is a plural 's', not if it's part of the noun ("class"). See https://www.thepunctuationguide.com/apostrophe.html#possessives

Now, while you were fixing this, you completely missed the fact that it should have been this:
```suggestion
 * DataPostprocessor class's documentation.
```
:-)

Reply:
You're completely right of course. Thanks for fixing it.